lib/globalize/active_record/migration.rb

```diff
@@ -72,13 +72,13 @@ def drop_translation_table!(options = {})
         # It's a problem because in early migrations would add all the translated attributes
         def complete_translated_fields
           translated_attribute_names.each do |name|
-            @fields[name] ||= column_type(name)
+            @fields[name] ||= (column_type(name) || :string)
```

Comment:
can `column_type` be the one to default to `:string`, if this is still needed, so that the caller doesn't need to know?

Reply:
As you can see, I changed the testing `column_type` to use the same `column_type` just to avoid code duplication, and for tests we need to be able return `nil` in some cases:
https://github.com/globalize/globalize/blob/b007dedab910ce4a1ceb6d071d2ea11a1bb717ae/test/globalize/migration_test.rb#L258-L260

 So, I thought, that it would be better if `column_type` returned either a type or `nil` if it's not existing for some reason.

Reply:
thanks for that!